src/modules/launcher/Plugins/Microsoft.Plugin.Indexer/SearchHelper/WindowsSearchAPI.cs

```diff
@@ -49,7 +50,12 @@ public List<SearchResult> ExecuteQuery(ISearchQueryHelper queryHelper, string ke
                 // # is URI syntax for the fragment component, need to be encoded so LocalPath returns complete path
                 // Using OrdinalIgnoreCase since this is internal and used with symbols
                 var string_path = ((string)oleDBResult.FieldData[0]).Replace("#", "%23", StringComparison.OrdinalIgnoreCase);
-                var uri_path = new Uri(string_path);
+
```

Comment:
Do we have a unit test for this. What is a invalid example

Reply:
> What is a invalid example

We don't know the string that can not be parsed for the user. But know we log the string if it can not be parsed.